### DISCUSSION TITLE
Thanks For Glorious 200

### DISCUSSION THREAD

**Discussion by nimsandu** [nimsandu/spicetify-bloom#143] [Announcements]
![THANKS](https://user-images.githubusercontent.com/57834844/210037905-89f66941-fba5-48ab-8210-de07625e8399.jpg)

It's been a long journey since we've asked to develop Bloom inside our repository instead of the "Spicetify-Themes" community repository. First, this theme was a complete bug other than being a friendly and calm user experience. Thanks to every Contributor who deserves every user's humble appreciation. This user experience has been built from the ground up. Every time we had an issue, someone volunteered to solve it in the nick of time. Today I'm glad to tell you that our journey has achieved another huge milestone with 200 stars on the repository. I've listed down every contribution as I can. Forgive me if I did forget any of you.

First, This Bloom theme is just a modification of the host theme "[spicetify-fluent](https://github.com/williamckha/spicetify-fluent) " by @williamckha. I won't forget this comment on my pr to Spicetify theme repository Sir.
![image](https://user-images.githubusercontent.com/57834844/210034267-6935eaaa-b4db-4962-95b8-a216ff6e0641.png)
it was my spirit on this whole journey. So, Mr. @williamckha accept our bloom sub family's humble appreciation.

Secondly, i would thanks @PINPAL for the first fix of Bloom #4 .
and also, thanks for the first issue #1 by @nigelonius.

at the movement (12.30.2022 04.34[GMT]) we have

- 8 open issues 
- 65 closed issues
- 0 open prs
- 65 closed prs
- 18 contributors
- 28 forks
- 200 stars

so next ill thanks every contributor for their unmatched help

- @MGPlayzYT                   (24 commits)
- @kyrie25                         (13 commits)
- @windowz414                (11 commits)
- @PINPAL                         (11 commits)
- @miukyo                         (08 commits)
- @lemonbigbig                (08 commits)
- @javalsai                         (06  commits)
- @Astromations               (05 commits)
- @BallOpener                  (03 commits)
- @Crushoverride007        (02 commits)
- accurics[bot]                    (01 commits)
- @sldless                          (01 commits)
- @theRealPadster             (01 commits)
- @q60                               (01 commits)
- @KAJdev                         (01 commits)
- @akashirai                      (01 commits)
- @syndek                         (01 commits)

Thank you! it was a lot for every bloomer including myself.

Next, I would love to thanks for every issue which helped this theme to expect a brighter future. every issue opener deserves this appreciation. 

Now, it's time to present my special thanks for some special contributors.

- @windowz414 
- @kyrie25 

Thanks for being long-term maintainers. Every time someone opened an issue, you both tried so hard to fix it by sacrificing your time.

and our 200th star is by @okkidwi thanks for helping us to reach this interesting milestone.

Every perfect journey relies on backstage heroes who sacrifice what they have to make it better. in this journey, our most Powerful strategy is that we are a family. "We are the Bloom family". our mission is to deliver something which would calm our community's complex mind.
and I would never be able to do that if I didn't have all your help.

All of you deserve my appreciation forever.
Thanks For being on this journey for this long.
Enjoy Bloom.
Nimsandu Kottage 
(30/12/2022)

**Comment by s1nnohlakes** [comment]
Glad I was part in making this theme better! And again, congratulations on reaching 200 stars!

**Comment by nimsandu** [comment]
Also, i would love to thanks @Dilith-Dahanayake for the first star. thank you milky.

**Comment by PINPAL** [comment]
Thankyou for making such a wonderful theme! Glad I could help out in some way :)

**Comment by Crushoverride007** [comment]
Thank you, and thank you again for making such a wonderful theme, glad i was of some help. looking forward to the future. 
Happy New Year guys 🥳

**Comment by nimsandu** [comment]
5 months later.......
we have 299 stargazers.
thanks a lot!

specially 

> @SunsetTechuila 
> @windowz414

**Reply by SunsetTechuila** [reply]
![image](https://user-images.githubusercontent.com/115353812/236641409-e514e573-0673-44d3-a59a-2672a250eeeb.png)

**Reply by nimsandu** [reply]
Thank you. I don't have words to thank you.

**Reply by nimsandu** [reply]
Together, this journey will continue. Thanks for unmatched 300.
